servers/media-server/src/server/connector/transports/mod.rs

```diff
@@ -0,0 +1,29 @@
+use async_trait::async_trait;
+
+pub mod nats;
+
+#[async_trait]
+pub trait ConnectorTransporter: Send + Sync {
+    async fn send(&self, data: &[u8]) -> Result<(), String>;
```

Comment:
We should not hard-coded that data transfer will be binary, instead we should parse Protocol object, then data serialize will be take care by protocol it-self.